Assets/MixedRealityToolkit.Examples/Demos/Solvers/Profiles/SurfaceMagnetismExampleMRTKConfigurationProfile.asset

```diff
@@ -0,0 +1,54 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
```

Comment:
Is there anything special here other than turning on SA? If not, I'd say let's just re-use "DefaultMixedRealityToolkitConfigurationProfile".

Reply:
In this profile, I made the mesh invisible to reduce visual noise.

Reply:
I'd say use the default and then add hand menu or toggle menu to switch out between occluded vs visible states in SA system

Reply:
Great idea!